src/Data/List/Ext.agda

```diff
@@ -1,12 +1,66 @@
 {-# OPTIONS --safe #-}
 module Data.List.Ext where
 
-open import Agda.Builtin.List using (List; []; _∷_)
-open import Agda.Builtin.Maybe using (Maybe; nothing; just)
-open import Agda.Builtin.Nat using (Nat; zero; suc)
```

Comment:
Not sure why we're using Builtins here.  It's generally preferred to use standard library definitions where possible.